backend/src/main/java/com/exadel/carinsurance/config/jwt/JwtAuthFilter.java

```diff
@@ -0,0 +1,72 @@
+package com.exadel.carinsurance.config.jwt;

```

Comment:
filter should not be located in config package.
Please use `config` package and it's subpackages only for classes related to project configuration such as bean definitions, utilities for computing data for bean initialization and etcetera. But not executable business code.

Reply:
Done